Resources/views/tagmanager_head.html.twig

```diff
@@ -4,6 +4,6 @@
 <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
         new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
         j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl+'{{ url_suffix }}';f.parentNode.insertBefore(j,f);
```

Comment:
@chrs-myrs @xyNNN I wonder if this is going to work without |raw filter (& is escaped and rendered as &+amp;) Can you please investigate? I think it is not usable right now.